.github/tests.yaml

```diff
@@ -0,0 +1,31 @@
+betelgeuse: "tests/foreman/ -v -m 'build_sanity'"
+broker[docker]: "tests/foreman/ -v -m 'build_sanity'"
+cryptography: "tests/foreman/ -v -m 'build_sanity'"
+deepdiff: "tests/foreman/endtoend/test_api_endtoend.py -v -k 'test_positive_get_links'"
+dynaconf[vault]: "tests/foreman/ -v -m 'build_sanity'"
+fauxfactory: "tests/foreman/ -v -m 'build_sanity'"
+flake8: "tests/foreman/ -v -m 'build_sanity'"
```

Comment:
None of the tests are likely to help here as this is a formatting tool and its not a direct dependency anywhere. Ignore it or automerge on pre-commit checks pass as its used there.